resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/submit/v1/Client.scala

```diff
@@ -99,6 +101,9 @@ private[spark] class Client(
 
   def run(): Unit = {
     logInfo(s"Starting application $kubernetesAppId in Kubernetes...")
+    loggingInterval.foreach { interval =>
+      require(interval > 0, "Logging interval must be a positive number.")
+    }
```

Comment:
seems like this check should be done inside the LoggingPodStatusWatcherImpl -- otherwise all clients of that API need to do this check